src/vs/workbench/api/common/extHostTerminalService.ts

```diff
@@ -161,8 +169,7 @@ export class ExtHostTerminal {
 			isExtensionCustomPtyTerminal: true,
 			icon: iconPath,
 			color: ThemeColor.isThemeColor(color) ? color.id : undefined,
-			isSplitTerminal,
-			target
```

Comment:
Don't we need target/location here?

Reply:
I don't think so bc location gets passed as `internalOptions`